models/qualtrics__directory.sql

```diff
@@ -68,10 +68,10 @@ agg_distributions as (
         count(distinct distribution.survey_id) as count_surveys_sent_30d
         
     from distribution_contact
-    join directory_contact 
+    join directory_contact
```

Comment:
Same comment about explicitly defining the type of join.

Reply:
updated to inner